.github/workflows/integration.yml

```diff
@@ -11,6 +11,10 @@ env:
   DATABASE_NAME: TestDatabase
   DATABASE_USERNAME: TestUsername
   DATABASE_PASSWORD: TestPassword
+  BUCKET_URL: https://ace-minio-1.loris.ca:9000
+  BUCKET_NAME: loris-rb-data
+  BUCKET_ACCESS_KEY: lorisadmin-ro
+  BUCKET_SECRET_KEY: Tn=qP3LupmXnMuc
```

Comment:
@maximemulder could this be secrets variables? I know this is for read-only data but still, not ideal to get credentials in the code.

Reply:
According to this doc:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#using-encrypted-secrets-in-a-workflow
> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.
So I guess our options are either:
- Keep the variables secret, but the integration tests must be triggered manually.
- Make the variables public.
Still looking into the above options, I may edit this message in the future.

Reply:
After the setting up of a bucket without keys taking too long to set up with IT, it was decided to hard-code the keys for now.